config/index.js

```diff
@@ -838,4 +844,8 @@ conf.set('smtp.verifySecondaryEmailUrl', conf.get('contentServer.url') + '/verif
 
 conf.set('isProduction', conf.get('env') === 'prod')
 
+if (conf.get('env') === 'prod') {
+  conf.set('snsTopicEndpoint', undefined);
```

Comment:
@vladikoff , is that ok or we need to "delete" it.

Reply:
@deeptibaghel let's do this check: `IF snsTopicEndpoint is set AND env is NOT dev THEN throw new Error('snsTopicEndpoint is only allowed in dev env')`

Reply:
This is done, now i think the credentials are needed.